Backend/dev/mock-servers/server.py

```
@@ -276,7 +468,7 @@ def main():
 
     server = HTTPServer(("0.0.0.0", args.port), MockHandler)
     log.info(f"Mock server running on :{args.port}")
-    log.info(f"APIs: POST/GET/DELETE /mock/status, POST/GET/DELETE /mock/override")
+    log.info(f"APIs: POST/GET/DELETE /mock/status, POST/GET/DELETE /mock/override, POST /mock/pass-status, POST /mock/scheduler/trigger")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)